src/Components/Actions/AddressAutocompleteAction/AddressAutocompleteAction.tsx

```diff
@@ -331,23 +264,32 @@ export const AddressAutocompleteAction: React.FC<AddressAutocompleteActionProps>
 
       <Spacer />
 
-      <ContinueButton
-        onClick={() => handleContinue(confirmedAddress!)}
-        disabled={!confirmedAddress}
-        text={(props.link || {}).label || 'Continue'}
-      />
-
-      <AddressAutocomplete
-        isActive={isAutocompleteActive}
-        onDismiss={() => setIsAutocompleteActive(false)}
-        selected={pickedSuggestion}
-        onSelect={handleSelectSuggestion}
-        onNotFound={handleNoAddressFound}
-        value={searchTerm}
-        onChange={handleChangeInput}
-        onClear={handleClearInput}
-        placeholder={props.placeholder}
-      />
+      {confirmedAddress ? (
+        <ContinueButton
+          onClick={() => handleContinue(confirmedAddress)}
+          disabled={false}
+          text={(props.link || {}).label || 'Continue'}
+        />
+      ) : (
+        <ContinueButton
+          onClick={() => {}}
+          disabled={true}
+          text={(props.link || {}).label || 'Continue'}
+        />
+      )}
```

Comment:
Maybe just make the button `disabled` if there's no `confirmedAddress` instead of conditionally rendering buttons with different attributes?
```suggestion
        <ContinueButton
          onClick={() => handleContinue(confirmedAddress)}
          disabled={Boolean(confirmedAddress)}
          text={(props.link || {}).label || 'Continue'}
        />
```

Reply:
Or just with implicit type coercion if you like that better 🙃 
```typescript
        <ContinueButton
          onClick={() => handleContinue(confirmedAddress)}
          disabled={confirmedAddress}
          text={(props.link || {}).label || 'Continue'}
        />
```

Reply:
The problem is that `confirmedAddress` is possibly null and therefore can't be passed directly to `handleContinue`.

We could use the "bang operator" to override this type check but I kind of prefer this explicit check instead. Where do you stand on that?

Reply:
Ah, I see, I missed that! Hm, I think non-null assertions should probably be used sparingly (so no,  don't have an exciting opinion there). But how about this?
```typescript
        <ContinueButton
          onClick={confirmedAddress ? () => handleContinue(confirmedAddress) : () => {}}
          disabled={confirmedAddress}
          text={(props.link || {}).label || 'Continue'}
        />
```
Or is that also messy? I just think it's a bit confusing with the "two" buttons.

Reply:
Right, I agree that this is more what you would expect. I can change to that.

Interesting to figure out a good way to handle this though 🤔 like if the function should accept `null` as well but throw a runtime exception if it ever get's called with a "null" value maybe?